learning_observer/learning_observer/rosters.py

```diff
@@ -360,15 +363,18 @@ def init():
     elif roster_source in ['test', 'filesystem']:
         ajax = synthetic_ajax
     elif roster_source in ["google_api"]:
-        ajax = google_ajax
+        ajax = combined_ajax
     elif roster_source in ["all"]:
         ajax = all_ajax
+    elif roster_source in ["canvas"]:
```

Comment:
Why not combine this with the `google_api` conditional since they both use the same function?
`elif roster_source in ["google_api", "canvas"]:`

Reply:
Fixed